### ISSUE TITLE
question:  Use yubikey with caddy

### ISSUE THREAD

**Issue by PeterB513** [greenpau/caddy-security#73]
I just want to use a yubikey to authenticate my login to a website using Caddy. Nothing more.  don’t need the other stuff which is presented in the plugin, sorry. Basically I am looking for a small and simple implementation. Is this possible with your plugin and what is an example of code of the Caddyfile. Thanks.

**Comment by greenpau** [comment]
@PeterB513 , thank you for the issue. I will try finding time answer your question this weekend.

We’re you able to set up auth with password and Yubico key? Do you have a working config?

**Comment by mouzzampk2014** [comment]
Yes that would be great. I would really appreciate if you can post Caddfile example for this. 

Thank you

**Comment by greenpau** [comment]
> Yes that would be great. I would really appreciate if you can post Caddyfile example for this.

@mouzzampk2014 , there is no special configuration for the use of Yubikey as a second factor. You just need to add it to your user profile. Please see Login with App Authenticator and Yubico U2F [Video](https://youtu.be/poOkq_jb1B0)

This specific issue is about making U2F token the only authentication requirement.
